orchagent/muxorch.h

```diff
@@ -133,6 +140,7 @@ struct NHTunnel
 typedef std::unique_ptr<MuxCable> MuxCable_T;
 typedef std::map<std::string, MuxCable_T> MuxCableTb;
 typedef std::map<IpAddress, NHTunnel> MuxTunnelNHs;
+typedef std::map<NextHopKey, std::string> NextHopTb;
```

Comment:
nit: `using`

Reply:
typedef is used throughout.. so keeping it for consistency